backend/docs/internal_v1_deployments.oai.yaml

```diff
@@ -0,0 +1,1130 @@
+openapi: 3.0.1
+info:
+  description: |
+    Internal API of deployments service
+  title: Deployments Internal API
+  version: "1"
+servers:
+  - url: http://mender-deployments:8080/api/internal/v1/deployments
+paths:
+  /health:
+    get:
+      operationId: Check Health
+      responses:
+        "204":
+          content: {}
+          description: |
+            Service is healthy and all dependencies are up and running.
+        "500":
+          content:
+            application/json:
+              example:
+                error: internal error
+                request_id: ffd712be-d697-4cb7-814b-88ff1e2eb5f6
+              schema:
+                $ref: "#/components/schemas/Error"
+          description: |
+            Unexpected internal error
+        "503":
+          content:
+            application/json:
+              example:
+                error: "error reaching MongoDB: context deadline exceeded"
+                request_id: ffd712be-d697-4cb7-814b-88ff1e2eb5f6
+              schema:
+                $ref: "#/components/schemas/Error"
+          description: |
+            Service unhealthy / not ready to accept traffic. At least one dependency is not running.
+      summary: Check the health of the service
+      tags:
+        - Internal API
```

Comment:
we are moving tags and summary to the end. while I am not adamant here, I got used to having it at the top. all the comments I make about the order of fields are based on what we already have in device auth open api3 -- I want to keep it consistent; we either change here or create a PR for the deviceauth.